src/main/scala/cromwell/engine/callactor/CallActor.scala

```diff
@@ -196,7 +196,7 @@ trait CallActor extends LoggingFSM[CallActorState, CallActorData] with CromwellA
     val message = executionResult match {
       case SuccessfulBackendCallExecution(outputs, executionEvents, returnCode, hash, resultsClonedFrom) =>
         WorkflowActor.CallCompleted(key, outputs, executionEvents, returnCode, if (workflowDescriptor.writeToCache) Option(hash) else None, resultsClonedFrom)
-      case SuccessfulFinalCallExecution => WorkflowActor.CallCompleted(key, Map.empty, Seq.empty, 0, None, None)
+      case SuccessfulFinalCallExecution => WorkflowActor.CallCompleted(key, Map.empty, Seq.empty, ScriptReturnCode(0), None, None)
```

Comment:
Does this work properly for a call with `continueOnReturnCode = 1`? 

It looks like it'll store 0 even if the "successful" return code was 1